priv/templates/bin

```diff
@@ -65,20 +65,31 @@ export BINDIR="$ERTS_DIR/bin"
 export EMU="beam"
 export PROGNAME="erl"
 export LD_LIBRARY_PATH="$ERTS_DIR/lib:$LD_LIBRARY_PATH"
-ERTS_LIB_DIR="$ERTS_DIR/../lib"
+SYSTEM_LIB_DIR="$(dirname "$ERTS_DIR")/lib"
```

Comment:
Aside from the rename, this mirrors

https://github.com/erlware/relx/blob/5a9f6ebbc30244d281d8e99aa3c9295b75efa975/priv/templates/extended_bin#L577

in the extended bin script.